Offroad-iOS/Offroad-iOS/Presentation/QuestQR/ViewControllers/QuestQRViewController.swift

```diff
@@ -22,25 +41,66 @@ class QuestQRViewController: UIViewController {
         view = questQRView
     }
     
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        setNavigationController()
+        setupNavigationControllerGesture()
+        setupCaptureSession()
+        setupDelegates()
+        setupPrevieLayer()
+        
+        DispatchQueue.global().async { [weak self] in self?.captureSession.startRunning() }
+    }
```

Comment:
이 코드 이렇게 작성하신 이유가 있으실까요? 설명 부탁드려욤🙏

Reply:
우선 이 코드에서 `captureSession`의 `startRunning()` 메서드는 카메라를 켜고 QR코드 인식를 시작하는 역할을 합니다!
그래서 `AVCaptureSession`의 `startRunning()` 메서드를 메인쓰레드에서 호출하니 background thread에서 호출하라고 경고가 뜨더라구요!
그래서 [공식 문서](https://developer.apple.com/documentation/avfoundation/avcapturesession)를 찾아봤더니, `startRunning()` 메서드는 UI responsive 하도록 비동기적으로 호출하라고 적혀있었습니다!
> **Important**
> The `startRunning()` method is a blocking call which can take some time, therefore start the session on a serial dispatch queue so that you don’t block the main queue (which keeps the UI responsive). See [AVCam: Building a camera app](https://developer.apple.com/documentation/avfoundation/capture_setup/avcam_building_a_camera_app) for an implementation example.

그래서 비동기적으로 호출하는 함수를 구현하였습니다!

(혹시 줄바꿈 없이 한 줄로 작성한 이유에 대해 물으신 거라면...코드의 길이가 그리 길지 않아 한 눈에 보기 편할 것이라 생각하여 저렇게 작성하였습니다! 줄바꿈을 하는 게 좋다고 생각하시면 자유로운 의견 주세요~)